src/components/MAPI/keypairs/ClusterDetailKeyPairs.tsx

```diff
@@ -137,101 +168,105 @@ const ClusterDetailKeyPairs: React.FC<IClusterDetailKeyPairsProps> = () => {
           pathname,
         }}
       >
-        <Box>
+        <>
```

Comment:
Unrelated, but `<Breadcrumb />` requires a single child, so this was displaying an error in the console